packages/slice-machine/src/hooks/useCurrentSlice.tsx

```diff
@@ -0,0 +1,42 @@
+import { selectCurrentSlice } from "@src/modules/selectedSlice/selectors";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import { SliceMachineStoreType } from "@src/redux/type";
+
+import { useRouter } from "next/router";
```

Comment:
you know the router state is in redux aswell, with operations coming from `connected-next-router`

Reply:
No I didn't know! Should I always use redux instead?